patterns/hero-overlapped-book-cover-with-links.php

```diff
@@ -0,0 +1,166 @@
+						<!-- /wp:spacer -->
+
+						<!-- wp:group {"className":"is-style-default","layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
+						<div class="wp-block-group is-style-default">
+
+							<!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"var:preset|spacing|20","left":"var:preset|spacing|20"}}}} -->
+							<div class="wp-block-columns">
+
+								<!-- wp:column {"verticalAlignment":"stretch"} -->
+								<div class="wp-block-column is-vertically-aligned-stretch">
+
+									<!-- wp:buttons {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"flex","orientation":"horizontal","flexWrap":"wrap","justifyContent":"space-between"}} -->
+									<div class="wp-block-buttons">
+
+										<!-- wp:button {"backgroundColor":"contrast","textColor":"base","width":100,"className":"is-style-outline","style":{"border":{"radius":"50px"},"typography":{"fontSize":"1rem"},"elements":{"link":{"color":{"text":"var:preset|color|base"}}},"spacing":{"padding":{"top":"1rem","bottom":"1rem"}}}} -->
+										<div class="wp-block-button has-custom-width wp-block-button__width-100 has-custom-font-size is-style-outline" style="font-size:1rem">
+											<a class="wp-block-button__link has-base-color has-contrast-background-color has-text-color has-background has-link-color wp-element-button" href="#" style="border-radius:50px;padding-top:1rem;padding-bottom:1rem">
+												<?php echo esc_html_x( 'Shop at Amazon', 'Hero - Overlapped book cover pattern button 1', 'twentytwentyfive' ); ?>
+											</a>
+										</div>
+										<!-- /wp:button -->
+
+										<!-- wp:button {"backgroundColor":"contrast","textColor":"base","width":100,"className":"is-style-outline","style":{"border":{"radius":"50px"},"typography":{"fontSize":"1rem"},"elements":{"link":{"color":{"text":"var:preset|color|base"}}},"spacing":{"padding":{"top":"1rem","bottom":"1rem"}}}} -->
+										<div class="wp-block-button has-custom-width wp-block-button__width-100 has-custom-font-size is-style-outline" style="font-size:1rem">
+											<a class="wp-block-button__link has-base-color has-contrast-background-color has-text-color has-background has-link-color wp-element-button" href="#" style="border-radius:50px;padding-top:1rem;padding-bottom:1rem">
+												<?php echo esc_html_x( 'Shop at Apple Books', 'Hero - Overlapped book cover pattern button 2', 'twentytwentyfive' ); ?>
+											</a>
+										</div>
+										<!-- /wp:button -->
+
+									</div>
+									<!-- /wp:buttons -->
+
+								</div>
+								<!-- /wp:column -->
+
+								<!-- wp:column {"verticalAlignment":"stretch"} -->
+								<div class="wp-block-column is-vertically-aligned-stretch">
+
+									<!-- wp:buttons {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"flex","orientation":"horizontal","flexWrap":"wrap","justifyContent":"space-between"}} -->
```

Comment:
I did not understand why the buttons use the outline style, but are then filled with the background color?
Can the default button design not be used here? The colors should not be needed, because they are the default, and adding them to the pattern makes it more difficult to use style variations.

Reply:
I thought it should look like the Figma design (just a black background, no hovering), but I changed the buttons to the default style.